services/site/src/pages/about.tsx

```diff
@@ -0,0 +1,220 @@
+import Footer from "app/components/Footer";
+import A11yphantLogoWithoutText from "app/components/icons/A11yphantLogoWithoutText";
+import Navigation from "app/components/Navigation";
+import { usePrefersReducedMotion } from "app/hooks/prefersReducedMotion";
+import clsx from "clsx";
+import Head from "next/head";
+import Image from "next/image";
+import React from "react";
+
+import a11yphantAnimation from "../../public/images/a11yphant-animation.gif";
+import a11yphantStatic from "../../public/images/a11yphant-static.jpg";
+
+const About: React.FunctionComponent = () => {
+  const prefersReducedMotion = usePrefersReducedMotion();
+
+  return (
+    <>
+      <Head>
+        <title>About | a11yphant</title>
+        <meta name="description" content="a11yphant is an interactive online course for web accessibility." />
+        {/* <!-- Facebook Meta Tags --> */}
+        <meta property="og:url" content="https://a11yphant.com" />
+        <meta property="og:type" content="website" />
+        <meta property="og:locale" content="en" />
+        <meta property="og:title" content="About" />
+        <meta property="og:description" content="a11yphant is an interactive online course for web accessibility." />
+        <meta property="og:image" content="https://a11yphant.com/images/mockups-social-media.jpg" />
+        <meta
+          property="og:image:alt"
+          content="A coding challenge in a11yphant with an instruction section, a code editor and a preview section to view the code you have just written."
+        />
+        {/* <!-- Twitter Meta Tags --> */}
+        <meta name="twitter:card" content="summary_large_image" />
+        <meta name="twitter:site" content="@a11yphant" />
+        <meta name="twitter:title" content="a11yphant" />
+        <meta name="twitter:description" content="a11yphant is an interactive online course for web accessibility." />
+        <meta name="twitter:image" content="https://a11yphant.com/images/mockups-social-media.jpg" />
+        {/* <!-- General Meta Tags --> */}
+        <meta name="theme-color" content="#121212" media="(prefers-color-scheme: dark)" />
+        <meta name="theme-color" content="#FFFFFF" media="(prefers-color-scheme: light)" />
+      </Head>
+      <Navigation />
+      <main className={clsx("h-full box-border max-w-screen-3xl mx-auto")}>
+        <div className={clsx("mx-8 py-8 h-main max-w-screen-3xl mt-12", "sm:mx-12 sm:mt-24", "lg:mx-24")}>
```

Comment:
Can you pls remove the mt-12 on this page + the privacy policy + legal notice pages? On mobile devices the spacing is just too big. I don't know why he set it to such a high value in the first place.